packages/cli/lib/services/dryrun.service.ts

```diff
@@ -75,11 +75,15 @@ class DryRunService {
             return;
         }
 
-        const providerConfigKey = config.find((config) => [...config.syncs, ...config.actions].find((sync) => sync.name === syncName))?.providerConfigKey;
+        let providerConfigKey = optionalProviderConfigKey;
 
         if (!providerConfigKey) {
-            console.log(chalk.red(`Provider config key not found, please check that the provider exists for this sync name: ${syncName}`));
-            return;
+            providerConfigKey = config.find((config) => [...config.syncs, ...config.actions].find((sync) => sync.name === syncName))?.providerConfigKey;
+
+            if (!providerConfigKey) {
+                console.log(chalk.red(`Provider config key not found, please check that the provider exists for this sync name: ${syncName}`));
```

Comment:
How does the user `check that the provider exists`?

Reply:
Added some copy with https://github.com/NangoHQ/nango/pull/2125/commits/4ab52803213d736e0b0e75067e55ff99511d16c9